RELEASENOTE.md

```diff
@@ -1,8 +1,34 @@
 Version Notes (Current Version: v1.3)
 =======================================
-v1.3 (Pre-release)
+v1.3
 -----------
-
+1. `docfx` improvements
+    1. Add subcommand `docfx template`. You can now `docfx template list` and `docfx template export -A` to list and export all the embeded templates!
```

Comment:
this is intentional, ` op/msdn/vs/docs` are for internal use and are planned to be removed from `docfx`.